### ISSUE TITLE
Manual grading: focus on score upon page load

### ISSUE THREAD

**Issue by jonatanschroeder** [PrairieLearn/PrairieLearn#6542]
When grading a question manually, for every new question a TA needs to focus on the question points to start grading. To avoid requiring that click, and presuming that TAs will typically type an answer (with possible feedback) and move on, the default focus should be on the manual points (or rubric once that's implemented).

**Comment by nwalters512** [comment]
Moving the focus on page load, e.g. with the `autofocus` attribute, can be bad for accessibility, especially for folks using screen readers: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus. That's not to say it should never be used, just that we should think carefully about if it's the best option here.

**Comment by echuber2** [comment]
Would it be a problem to make this an opt-in setting? (Local storage?)

**Comment by jonatanschroeder** [comment]
With rubrics in place and strongly recommended, and having keyboard shortcuts available for most inputs and to submit the score, I think this is not as valuable anymore. A user can easily use keyboard shortcuts for rubric items and feedback without the need to change focus on start. Closing as not planned.
